vm/src/obj/objlist.rs

```diff
@@ -96,6 +97,30 @@ impl PyList {
 
         start..stop
     }
+
+    pub fn get_byte_inner(&self, vm: &VirtualMachine) -> PyResult<objbyteinner::PyByteInner> {
+        let mut elements = Vec::<u8>::with_capacity(self.get_len());
+        for elem in self.elements.borrow().iter() {
+            let method = vm.get_method_or_type_error(elem.clone(), "__int__", || {
+                format!(
+                    "'{}' object cannot be interpreted as an integer",
+                    elem.class().name
+                )
+            })?;
+            let result = vm.invoke(&method, PyFuncArgs::default())?;
+            if let Some(result) = result.payload::<PyInt>() {
+                match result.as_bigint().to_u8() {
```

Comment:
It's ignored if `__int__` doesn't return an `int`, maybe use `PyIntRef::try_from_object`?

Reply:
Thank you for reviewing. I fixed my code. Also current code has bug. Because current code cannot handle `int.from_bytes([255.0, 0, 0], 'big')` as error. I fixed it, too.